openedx/core/djangoapps/discussions/views.py

```diff
@@ -92,6 +138,19 @@ def get(self, request, course_key_string, **_kwargs) -> Response:
         serializer = self.Serializer(configuration)
         return Response(serializer.data)
 
+    def post(self, request, course_key_string, **_kwargs) -> Response:
+        """
+        Handle HTTP/POST requests
+
+        TODO: Should we cleanup orphaned LTI config when swapping to cs_comments_service?
+        """
+        course_key = self._validate_course_key(course_key_string)
+        configuration = DiscussionsConfiguration.get(course_key)
```

Comment:
What happens if a DiscussionsConfiguration doesn't yet exist for that course_key?  Or are we pre-supposing that one already exists from 1) a data migration of existing courses and 2) creating a DiscussionsConfiguration when the course was created?

Reply:
Good catch: That helper returns a "shell" object if no database record exists yet, basically:

```python
return DiscussionsConfiguration(context_key=course_key, enabled=False)
```

So we can still call `.save()`  on it (and/or its serializer) and Django should figure out automagically whether to create or update.